src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -5,6 +5,7 @@ Release History
 
 2.0.24
 ++++++
+* vmss: (PREVIEW) cross zone support
```

Comment:
We should call out the breaking behavior here.

Reply:
Done